packages/react-components/src/components/StartRealTimeTextButton.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/* @conditional-compile-remove(rtt) */
+import { ControlBarButton, ControlBarButtonProps } from './ControlBarButton';
+/* @conditional-compile-remove(rtt) */
+import React from 'react';
+/* @conditional-compile-remove(rtt) */
+import { _HighContrastAwareIcon } from './HighContrastAwareIcon';
+/* @conditional-compile-remove(rtt) */
+import { useLocale } from '../localization';
+
+/* @conditional-compile-remove(rtt) */
+/**
+ * @beta
+ */
+export interface StartRealTimeTextButtonProps extends ControlBarButtonProps {
+  /**
+   * Utility property for using this component with communication react handlers
+   * Start RealTimeText based on RealTimeText state
+   */
+  onStartRealTimeText: () => Promise<void>;
+  /**
+   * If RealTimeText is on
+   */
+  isRealTimeTextOn: boolean;
+  /**
+   * Optional strings to override in component
+   */
+  strings?: StartRealTimeTextButtonStrings;
+}
+
+/* @conditional-compile-remove(rtt) */
+/**
+ * Strings for the hold button labels
+ * @beta
+ */
+export interface StartRealTimeTextButtonStrings {
+  /**
+   * Label for when action is to start RealTimeText
+   */
+  onLabel: string;
+  /**
+   * Content for when button is checked, RealTimeText is on
+   */
+  tooltipOnContent: string;
```

Comment:
is there any strings for Aria we need to add?

Reply:
No we don't need to. It's using the control bar buttons